panel/viewable.py

```diff
@@ -277,6 +277,25 @@ def _repr_mimebundle_(self, include=None, exclude=None):
         if not loaded and 'holoviews' in sys.modules:
             import holoviews as hv
             loaded = hv.extension._loaded
+
+        if config.jupyter_ext == 'ipywidgets':
```

Comment:
Maybe `config.comms`?  It's more than just comms (says @philippjfr), but `pn.extension(comms='ipywidgets')` reads much more nicely than `pn.extension(jupyter_ext='ipywidgets')`.